samples/CoreAnimationTest/CoreAnimationTest/PropertyTableViewCell.h

```diff
@@ -0,0 +1,25 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <UIKit/UIKit.h>
+
+@interface PropertyTableViewCell : UITableViewCell
+
+@property UILabel* titleLabel;
+
+- (void)setDisabled:(BOOL)disabled;
```

Comment:
It's subjective - since it's more effort to rename all the other cell versions, I'll keep as-is.

---
In reply to: [88709311](https://github.com/Microsoft/WinObjC/pull/1406#discussion_r88709311) [](ancestors = 88709311)